<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LauncherWindow</class>
 <widget class="QMainWindow" name="LauncherWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>575</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OpenApoc Launcher</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>791</width>
      <height>511</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="settingsTab">
     <attribute name="title">
      <string>Settings</string>
     </attribute>
     <widget class="QWidget" name="gridLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>781</width>
        <height>481</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="2" column="1">
        <widget class="QCheckBox" name="fullscreenCheckBox">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLineEdit" name="customResolutionX">
           <property name="inputMask">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="customResolutionByLabel">
           <property name="text">
            <string>x</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="customResolutionY">
           <property name="inputMask">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="fullscreenLabel">
         <property name="text">
          <string>Fullscreen</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="resolutionLabel">
         <property name="text">
          <string>Resolution</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="resolutionBox"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="languageLabel">
         <property name="text">
          <string>Language</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="customResolutionLabel">
         <property name="text">
          <string>Custom Resolution</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="languageBox"/>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="pathsTab">
     <attribute name="title">
      <string>Paths</string>
     </attribute>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>771</width>
        <height>481</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="2" column="1">
        <widget class="QLineEdit" name="dataPath"/>
       </item>
       <item row="2" column="3">
        <widget class="QPushButton" name="browseDataDir">
         <property name="text">
          <string>Browse</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="cdPath"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="cdPathLabel">
         <property name="text">
          <string>CD Path</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QPushButton" name="browseCDFile">
         <property name="text">
          <string>Browse file</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="dataPathLabel">
         <property name="text">
          <string>Data Path</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="browseCDDir">
         <property name="text">
          <string>Browse directory</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="modsTab">
     <attribute name="title">
      <string>Mods</string>
     </attribute>
     <widget class="QListWidget" name="enabledModsList">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>351</width>
        <height>192</height>
       </rect>
      </property>
      <property name="dragEnabled">
       <bool>true</bool>
      </property>
      <property name="dragDropMode">
       <enum>QAbstractItemView::InternalMove</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>251</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Enabled mods</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>250</y>
        <width>251</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Disabled mods</string>
      </property>
     </widget>
     <widget class="QPushButton" name="enableButton">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>50</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Enable</string>
      </property>
     </widget>
     <widget class="QPushButton" name="disableButton">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>80</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Disable</string>
      </property>
     </widget>
     <widget class="QLabel" name="nameLabel">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>50</y>
        <width>59</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>20</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Selected mod info</string>
      </property>
     </widget>
     <widget class="QLabel" name="modName">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>70</y>
        <width>271</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>modName</string>
      </property>
     </widget>
     <widget class="QLabel" name="authorLabel">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>90</y>
        <width>59</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Author</string>
      </property>
     </widget>
     <widget class="QLabel" name="modAuthor">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>110</y>
        <width>271</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>modAuthor</string>
      </property>
     </widget>
     <widget class="QLabel" name="linkLabel">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>130</y>
        <width>59</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Link</string>
      </property>
     </widget>
     <widget class="QLabel" name="modLink">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>150</y>
        <width>271</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>modLink</string>
      </property>
     </widget>
     <widget class="QLabel" name="descriptionLabel">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>210</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Description</string>
      </property>
     </widget>
     <widget class="QLabel" name="modDescription">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>230</y>
        <width>271</width>
        <height>131</height>
       </rect>
      </property>
      <property name="text">
       <string>modDescription</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QListWidget" name="disabledModsList">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>270</y>
        <width>351</width>
        <height>211</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="versionLabel">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>170</y>
        <width>59</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Version</string>
      </property>
     </widget>
     <widget class="QLabel" name="modVersion">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>190</y>
        <width>271</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>modVersion</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="playButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>530</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Play</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exitButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>530</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
